x-pack/legacy/plugins/uptime/public/components/functional/integration_group.tsx

```diff
@@ -75,30 +76,30 @@ export const IntegrationGroup = ({
           />
         </EuiFlexItem>
       ) : null}
-      {isInfraAvailable ? (
+      {isMetricsAvailable ? (
         <React.Fragment>
           <EuiFlexItem>
             <IntegrationLink
               ariaLabel={i18n.translate(
                 'xpack.uptime.monitorList.infraIntegrationAction.ip.ariaLabel',
```

Comment:
Did you intentionally not modify these translation keys? If so I am ++ on that, because we won't need to get the same values re-translated for new keys.